less/modifiers.less

```diff
@@ -61,10 +61,14 @@ html[data-color-profile=default] {
 /**
  * Remove img tags with aria-hidden
  * Remove theme background images
+ * Remove svg player with aria-hidden (adapt-svg)
+ * Remove canvas player with aria-hidden (adapt-graphicLottie)
  */
 .a11y-no-background-images {
   img[aria-hidden=true] { display: none }
   .has-bg-image { background-image: none !important; }
+  .svg__player[aria-hidden=true] { display: none; }
+  .graphiclottie__player[aria-hidden=true] { display: none; }
```

Comment:
I'm really not sure about referencing other plugins from here. Will need to look at this in more detail at a later stage to make is less tightly coupled.

Reply:
I wasn't sure about this myself. We do have other instances where plugins require fixes for Visua11y support such as Narrative https://github.com/cgkineo/adapt-visua11y/issues/50. I'm sure more will come to light too.
I decided to keep the amends within Visua11y modifiers.less to keep any plugin modifications in the same place. We already have [.has-bg-image](https://github.com/cgkineo/adapt-visua11y/blob/2289efe9e31fa437b5dd8a1f08c0f703b27c4b25/less/modifiers.less#L69) which is specific to Vanilla. Whether we keep core plugin modifications in here and non-core plugins contain their own modifications is an an option.
For adapt-scrollsnap, it's not on github but I know in projects we've often included a component.less file to contain all the component overrides that are required for builds using scrollsnap functionality, this could be an option too.